### ISSUE TITLE
Error when python main.py (name error)

### ISSUE THREAD

**Issue by abeI42** [agumonlyt/Bumblebee#1]
""NameError: name 'interception' is not defined. Did you mean: 'Interception'?""

anyone can help me with this?

**Comment by agumonlyt** [comment]
yes. have you installed interception? what is your cmd output when you run?
![image](https://github.com/agumonlyt/maplestorybot/assets/127786360/4dec0ced-2e47-468b-a82c-d9115870d2d6)

**Comment by R0ksta** [comment]
I am having the same issue. I have uninstalled and installed 'Interception' twice (both times its installing correctly and asking for reboot). 

Step 7, 'cd Bumblebee' doesn't exist in the folder structure (so I skipped this)

Step 8, I installed in the main folder directory

Step in 9, provides me the following error:

![image](https://github.com/agumonlyt/maplestorybot/assets/173223815/47f737a3-4aa3-4700-8658-d237128869d0)

**Comment by sufian-iskandar** [comment]
> yes. have you installed interception? what is your cmd output when you run? ![image](https://private-user-images.githubusercontent.com/127786360/324918528-4dec0ced-2e47-468b-a82c-d9115870d2d6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkzMjkwMzIsIm5iZiI6MTcxOTMyODczMiwicGF0aCI6Ii8xMjc3ODYzNjAvMzI0OTE4NTI4LTRkZWMwY2VkLTJlNDctNDY4Yi1hODJjLWQ5MTE1ODcwZDJkNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyNVQxNTE4NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iYjIyYTk0ZmE2NjAxYWYyMzZkYTFmYmI2N2NiMWUxZTNjYmJlMmY2NmFjMjUyNTMxMDAyNjA1M2ZmN2U3ZGY1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.uyeSH2-ljEYhGRP1QHEuVIvUNK0EQAXAETKVQSLIyAk)

how do i set my maplestory resolution in main.py?
